js/task/watch.js

```diff
@@ -28,6 +33,16 @@ module.exports = (styleguide, gulp) => {
   styleguide.watch.js = () => {
     gulp.watch('styleguide/**/*.js', { cwd: styleguide.path.root() }, deps(styleguide.task.js()))
       .on('change', onChange)
+    // waiting for change in the minified file before copying unminified to minified
+    if (process.env.JS_MIN !== 'true') {
+      gulp.watch('styleguide/All.min.js', { cwd: styleguide.path.build() })
+        .on('change', function (event) {
+          logger.info(`COPY .js >> .min.js: ${event.path}`)
```

Comment:
I think this log message should be removed because it makes the watcher log too verbose.